sqlglot/optimizer/simplify.py

```diff
@@ -729,31 +729,43 @@ def _simplify_binary(expression, a, b):
     return None
 
 
-def simplify_parens(expression):
+def simplify_parens(expression: exp.Expression, dialect: DialectType) -> exp.Expression:
     if not isinstance(expression, exp.Paren):
         return expression
 
     this = expression.this
     parent = expression.parent
     parent_is_predicate = isinstance(parent, exp.Predicate)
 
+    if isinstance(this, exp.Select):
+        return expression
+
+    if isinstance(parent, (exp.SubqueryPredicate, exp.Bracket)):
+        return expression
+
+    # Handle risingwave struct columns
+    # see https://docs.risingwave.com/sql/data-types/struct#retrieve-data-in-a-struct
     if (
-        not isinstance(this, exp.Select)
-        and not isinstance(parent, (exp.SubqueryPredicate, exp.Bracket))
-        and (
```

Comment:
Perhaps we can keep the old conjunction and add a new term like `isinstance(parent, exp.Dot)`. We may not even have to factor in the dialect for this, seems like an innocuous change.